src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/messaging/PrisonerReceivedEventDueToTransferTest.kt

```diff
@@ -24,80 +24,22 @@ class PrisonerReceivedEventDueToTransferTest : IntegrationTestBase() {
   }
 
   @Test
-  fun `should update Review Schedule and send outbound message given 'prisoner received' (transfer) event for prisoner that has an active Review Schedule with review date later than adjusted date`() {
+  fun `should update Review Schedule and send outbound message given 'prisoner received' (transfer) event for prisoner that has an active Review Schedule`() {
     // Given
     // an induction and action plan are created. This will have created the initial Review Schedule with the status SCHEDULED
     val prisonNumber = setUpRandomPrisoner()
     createInduction(prisonNumber, aValidCreateInductionRequestForPrisonerNotLookingToWork(prisonId = ORIGINAL_PRISON))
     createActionPlan(prisonNumber)
 
-    val expectedLatestReviewDate = LocalDate.now().plusDays(10)
-    // Set the latest review date to be later than the adjusted date that a transfer might set
-    updateReviewScheduleRecordLatestReviewDate(prisonNumber, expectedLatestReviewDate)
-
-    // The above calls set the data up but they will also generate events so clear these out before starting the test.
-    // Before clearing the queues though we need to wait until the "plp.review-schedule.updated" event on the REVIEW queue is received.
-    await untilCallTo {
-      reviewScheduleEventQueue.countAllMessagesOnQueue()
-    } matches { it != null && it > 0 }
-    clearQueues()
-
-    val sqsMessage = aValidHmppsDomainEventsSqsMessage(
+    // Set the review dates so that we can assert that they've been changed
+    updateReviewScheduleReviewDates(
       prisonNumber = prisonNumber,
-      eventType = PRISONER_RECEIVED_INTO_PRISON,
-      additionalInformation = aValidPrisonerReceivedAdditionalInformation(
-        prisonNumber = prisonNumber,
-        prisonId = PRISON_TRANSFERRING_TO,
-        reason = TRANSFERRED,
-      ),
+      earliestReviewDate = LocalDate.now().plusMonths(1),
```

Comment:
This is essentially the scenario/bug we are fixing , where the existing ReviewSchedule has an earliest and latest date set in the future sometime.
In the current impl (before this PR) when someone transfers we only set the latestReviewDate to today+10, so in this case the review schedule would have an earliest date of today+6 months and a latest review date of today+10 days.

This PR fixes this problem, and the changes to this test setup the scenario on this basis and assert **both** the earliest and latest dates have been changed